### PR TITLE
Rental mapper implemented

### PR DESCRIPTION
**Pull request by CodeWebWeaver**
implement Rental DTO mapper for RentalService

### CODE REVIEW

File: src/main/java/com/jvavateam/carsharingapp/controller/RentalController.java [hunk 1]
```diff
@@ -31,28 +32,28 @@ public class RentalController {
...
     @Operation(summary = "Get user rental by id",
             description = "Get specific rental")
-    public RentalResponseDto get(@PathVariable Long id) {
+    public CreateRentalResponseDto get(@PathVariable Long id) {
         return rentalService.getById(id);
     }
 
     @PostMapping("/rentals/{id}/return")
     @Operation(summary = "Update rental return date",
             description = "Set actual return date (increase car inventory by 1)")
```
Comment (IvanFadieiev) [inline src/main/java/com/jvavateam/carsharingapp/controller/RentalController.java:53]: Consider add some Response status, but it's up to you)

File: src/main/java/com/jvavateam/carsharingapp/mapper/RentalMapper.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+@Mapper(config = MapperConfiguration.class)
+public interface RentalMapper {
+    @Mapping(target = "carId", source = "car.id")
+    @Mapping(target = "userId", source = "user.id")
+    CreateRentalResponseDto toCreateDto(Rental rental);
+
+    @Mapping(target = "carId", source = "car.id")
+    @Mapping(target = "userId", source = "user.id")
+    RentalReturnResponseDto toReturnDto(Rental rental);
```
Comment (IvanFadieiev) [inline src/main/java/com/jvavateam/carsharingapp/mapper/RentalMapper.java:19]: It's better to ignore rental id in response DTO

**Review by IvanFadieiev — CHANGES_REQUESTED** [review]

File: src/main/java/com/jvavateam/carsharingapp/controller/RentalController.java (hunk 1 above)
Comment (Dimagaa) [reply src/main/java/com/jvavateam/carsharingapp/controller/RentalController.java:53]: Agree, Response.Status.Created would be great!

File: src/main/java/com/jvavateam/carsharingapp/mapper/RentalMapper.java (hunk 2 above)
Comment (CodeWebWeaver) [reply src/main/java/com/jvavateam/carsharingapp/mapper/RentalMapper.java:19]: No, I disagree we need ID in response

**Pull request closed without merging**